osd/maintenance_completed.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Info",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "{$DATE} Cluster maintenance completed",
+    "description": "{$DATE} cluster maintenance is now completed.",
```

Comment:
add "successfully"
and add that if the customer notices an issue they can open a support ticket

Reply:
Done, can you please verify?